data/json/items/melee/bludgeons.json

```diff
@@ -242,6 +242,25 @@
     "to_hit": 1,
     "category": "weapons"
   },
+  {
+    "id:": "plunger_weapon",
+    "name": { "str": "brain cleaner" },
+    "type": "TOOL",
+    "description": "A barbaric war pick made by driving a metal spike into a toilet plunger. World War IV will be fought with sticks and stones...",
```

Comment:
This is excessively specific, it's a nearly uselessly lightweight base item, and the "drive the spike through it" approach is more likely than not to either break or fail to affix the spike altogether.
The rest of the additions are fine, but I'd rather pass on this one.

Reply:
Understood and agreed. My main motivation was making the plunger do something. I feel less guilt over adding a useless clutter item now that I know rubber is going to be a distinct material from plastic, since hey, there's a consistent source now!